backend/src/paymaster/index.ts

```diff
@@ -139,7 +139,7 @@ export class Paymaster {
           preVerificationGas: BigNumber.from(packedUserOp.preVerificationGas).toHexString(),
           verificationGasLimit: BigNumber.from(userOp.verificationGasLimit).toHexString(),
           callGasLimit: BigNumber.from(userOp.callGasLimit).toHexString(),
-          paymasterVerificationGasLimit: BigNumber.from(30000).toString(),
+          paymasterVerificationGasLimit: BigNumber.from(30000).toHexString(),
```

Comment:
Is this 30000 hardcoded in the contract as well? If so, can we make this as a configurable variable with a default value as 30,000?

Reply:
okay